tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -281,6 +299,49 @@ private void handleRunTests(List<String> toolNames, List<String> sources) {
         }
     }
 
+
+    private void setUpGa4Ghv20Api() {
+        this.tooltesterConfig = new TooltesterConfig();
+        io.dockstore.openapi.client.ApiClient defaultApiClient = io.dockstore.openapi.client.Configuration.getDefaultApiClient();
+        defaultApiClient.setBasePath(this.tooltesterConfig.getServerUrl());
+        setGa4Ghv20Api(new Ga4Ghv20Api(defaultApiClient));
+    }
+
+    private void runToolTesterOnWorkflows() throws InterruptedException {
+        List<WorkflowRunner> workflowsToRun = new ArrayList<>();
+        setUpGa4Ghv20Api();
+
+        // These are just left here for testing purposes, until I find the API commands to get them
+        workflowsToRun.add(new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "Dockstore.json"));
+        workflowsToRun.add(new WorkflowRunner("github.com/dockstore-testing/wes-testing/agc-fastq-read-counts", "main", "/agc-examples/fastq/input.json", getGa4Ghv20Api()));
+
+        for (WorkflowRunner workflow : workflowsToRun) {
+            workflow.runWorkflow();
+            sleep(WAIT_TIME);
```

Comment:
If we make too many requests to agc too quickly, it will cause agc to thrown an error.

Reply:
Current, it is set to wait 10 seconds, this seems to work. Depending on the size of the workflows we are running, it might be wise to make the wait time a bit longer.

Reply:
> If we make too many requests to agc too quickly, it will cause agc to thrown an error.

what's the error? Waiting is totally fine, but a little curious if this is a known issue and in any case it would be good to have a github issue with the error message (for reference for end-users)

Reply:
It looks like there is no longer an error.

I think it was caused by some other error that I have now fixed.

I have now removed the wait time for running the workflows. Although I am still having it wait when requesting the logs, as there is no point in continually asking.